pkg/bench/get.go

```diff
@@ -186,8 +186,12 @@ func (g *Get) Start(ctx context.Context, wait chan struct{}) (Operations, error)
 				opts.VersionID = obj.VersionID
 				o, err := client.GetObject(ctx, g.Bucket, obj.Name, opts)
 				if err != nil {
-					console.Errorln("download error:", err)
-					op.Err = err.Error()
+					// In-flight requests may get canceled due to
+					// auto-term or reaching the `--duration` specified
+					if !isErrContext(err) {
+						console.Errorln("download error:", err)
+						op.Err = err.Error()
```

Comment:
This is wrong. It will be counted as a completed request which will skew numbers.

Switch to a non-terminating context like the others.  I will make a PR.